### DISCUSSION TITLE
What’s the best config for Enigma2 and SatPI working together?

### DISCUSSION THREAD

**Discussion by patacalas** [Barracuda09/SATPI#159] [Q&A]
Hi @Barracuda09,
first of all, thanks for SatPI. I use it everyday and it's great.

I have a GigaBlue UHD Quad 4K with FBC tuners connected to an Unicable system (Sat A Astra, Sat B Hotbird, Sat C Hispasat) and I'm trying to improve my system settings, because I'm using Enigma2 (OpenVIX) and SatPI together and I get some issues.

Is there any way to disable tuners in Enigma? other than setting disabled on Enigma tuner settings.

I'm currently using tuners B, F, G, H (frontends 1, 5, 6, 7) with SatPI and tuners A, C, D, E with Enigma2, as I use them for tv recording on internal HDD.

Firstly, I tried to disable Enigma's tuner B to avoid problems, but it's impossible to disable FBC tuners F, G, H (you can only choose between FBC automatic or FBC advanced, not disabled option). Obviously, I set FBC Connection B in SatPI enabled frontends and send Diseqc via linked root tuner.
It works almost fine, but sometimes SatIP channels freeze when zapping in Enigma.

After a lot of tries, I discovered that using a signal splitter to connect both tuner A and B (instead of internal loop) gives me better results. Moreover, I enabled all tuners in Enigma and set tuner priority. If I set priority to tuner A, Enigma will use tuners in the following order: A > C > D > E > B > F > G > H. In that way, I can watch and record up to 4 channels in Enigma2 without problems, and SatIP works fine too, but I would like to disable SatPI tuners to make sure Enigma2 can't use them.

Do you know other system settings for Enigma and SatPI working fine together?

Thanks in advance for your response.
Regards

**Comment by Barracuda09** [comment]
Hi @patacalas 

Sorry, to be honest, I'm no expert with Enigma. I don't really use it myself. And I do not have a screen connected to the boxes.

- So it is not possible to disable a (FBC)frontend in Enigma? I believe with 'normal' frontend it is possible to disable one?. 
- The Gigablue internal loop-through still has some issues (I feel/think). So an external is a nice solution.

Or do you mean something else with disable, or is it disabling for the whole FBC tuner (frontend A -H)?

Kind regards,

Marc
